jupyterlab_pullrequests/managers/manager.py

```diff
@@ -112,3 +116,75 @@ def read_notebook(content):
         )
 
         return {"base": prev_nb, "diff": thediff}
+
+    async def _call_service(
+        self,
+        url: str,
+        load_json: bool = True,
+        method: str = "GET",
+        body=None,
+        params: Optional[Dict[str, str]] = None,
+        headers: Optional[Dict[str, str]] = None,
+    ):
+        if not self._access_token:
+            raise tornado.web.HTTPError(
+                status_code=http.HTTPStatus.BAD_REQUEST,
+                reason="No access token specified.",
```

Comment:
as this gets pushed up to the frontend, perhaps make less opaque with `"No access token specified. Please set GithubConfig.github_access_token in jupyter_server_config"`